frontend/src/app/app.module.ts

```diff
@@ -87,6 +87,11 @@ import { ChargeTypeResolverService } from './charge-type-resolver.service';
 import { ChargeService } from './charge.service';
 import { ChargesResolverService } from './charges-resolver.service';
 import { PersonChargeEditComponent } from './person-charge-edit/person-charge-edit.component';
+import { registerLocaleData } from '@angular/common';
+import localeFr from '@angular/common/locales/fr';
+import { EmptyResponseBodyErrorInterceptorService } from './empty-response-body-error-interceptor.service';
+
+registerLocaleData(localeFr);
```

Comment:
Yes, since we only have one language supported, and since we already customize the locale by hand, I thought it would be easier and faster to leave the build simple, and to hard-code everything to French.